### ISSUE TITLE
IC2 Recipes with damage based items break after shift clicking in Pattern Terminal (1.7.10)

### ISSUE THREAD

**Issue by ThePixelbrain** [vfyjxf/NotEnoughEnergistics#49]
This is an odd one and it took me quite a while to figure out it's NEE.

**Description:**
I'm unable to craft any IC2 items that take damage based ingredients after shift clicking a recipe with a damage based ingredient into the ME Pattern Terminal.
A good example of this is the MFSU that takes Lapotron Crystals as an input. I can craft the MFSU with the damaged Lapotron Crystals fine until I shift click the MFSU recipe into the Pattern Terminal. The recipe shown in NEI only allows non-damaged Lapotron Crystals (unobtainable in survival). After I do this once, I'm unable to craft them with any damaged Lapotron Crystals at all until I restart my game. The non-damaged Lapotron Crystals still work fine.
This issue only occurs with NEE installed.

**Steps to reproduce:**
1. Get the items required for a MFSU. Use the Lapotron Crystals that aren't charged (Damage value 26).
2. Put the items required to craft a MFSU into any crafting table. You can see that the recipe will output a MFSU.
3. Shift click the MFSU recipe into a Pattern Terminal. This will show a recipe with Lapotron Crystals without a damage value.
4. Look into the crafting table again. You'll see that the MFSU recipe is now invalid.
5. You can however still craft the recipe with Lapotron Crystals without a damage value (unobtainable in survival).

**Test environment**:
- Minecraft 1.7.10
- Forge 10.13.4.1614
- Tested in SP, not tested in MP
- Installed mods:
  - Applied Energistics 2 RV3 Beta 6
  - CodeChickenCore 1.0.7.48
  - IndustrialCraft 2 2.2.827
  - NotEnoughEnergistics 1.4.2
  - NotEnoughItems 1.0.5.120
  - Waila 1.5.10

tl;dr: A fresh 1.7.10 instance with everything latest and only NEI, AE2, IC2 and NEE.

**Additional notes**:
I made a short video of the issue for easier understanding: https://scs.pixelbrain.dev/QXBsnqaK.mp4
